auction-server/src/auction/service/verification.rs

```diff
@@ -331,6 +334,7 @@ pub struct BidDataSvm {
     pub deadline:                        OffsetDateTime,
     pub submit_type:                     SubmitType,
     pub express_relay_instruction_index: usize,
+    pub user_wallet_address:             Option<Pubkey>,
```

Comment:
i think this should live within the SubmitType enum

Reply:
This doesn't work super well because SubmitType is also created in handle_auction and I'd rather not refactor it there